### ISSUE TITLE
ImportError: No module named flufl.lock

### ISSUE THREAD

**Issue by johannesring** [hashdist/hashstack#911]
Consider the following in `default.yaml`:
```yaml
extends:            
- file: linux.yaml  
                    
packages:           
  python:           
    link: shared    
  flufl_lock:       
```
After `hit build` I try to import in Python:
```console
$ ./default/bin/python -c "import flufl.lock"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ImportError: No module named flufl.lock
```
The problem is that there is a missing `__init__.py` in `default/lib/python2.7/site-packages/flufl`. This file is installed if I do a normal `python setup.py install`. Why is this file not installed when using HashDist?

One fix I found was to patch `setup.py` by removing the line with `namespace_packages=['flufl'],` (see patch below). Then the `__init__.py` file is installed correctly also by HashDist, but there must be another way to fix this?

```diff
diff -Nru flufl.lock-2.3.1.orig/setup.py flufl.lock-2.3.1/setup.py            
--- flufl.lock-2.3.1.orig/setup.py      2016-01-19 11:54:13.423817040 +0100   
+++ flufl.lock-2.3.1/setup.py   2016-01-19 11:54:47.999830147 +0100           
@@ -27,7 +27,6 @@                                                             
 setup(                                                                       
     name='flufl.lock',                                                       
     version=__version__,                                                     
-    namespace_packages=['flufl'],                                            
     packages=find_packages(),                                                
     include_package_data=True,                                               
     maintainer='Barry Warsaw',                                               
```

**Comment by ahmadia** [comment]
Namespace packages are a pain. I think we have handled this in other
installs by being smarter about how we build and install. If I recall
correctly, setup tools needs to do something funny with a path file to make
it all work.

**Comment by johannesring** [comment]
Thanks @ahmadia. I went through all the yaml files that use `setuptools_package`, but didn't find anything that could be used to fix this problem. However, I now have another workaround that I think is acceptable: simply copy the missing `__init__.py` file manually into the correct location after installation. I will make a pull request.

**Comment by johannesring** [comment]
Fixed in #912.
